DOCS-#3954: Add Dask example notebooks (#4139)
modin-project/modin: examples/tutorial/jupyter/README.md

> # Jupyter notebook examples to run Modin on PandasOnRay or PandasOndask
> 

Reviewer: ```suggestion
# Jupyter notebook examples to run with Modin

Currently we provide tutorial notebooks for the following execution backends:

- [PandasOnRay](https://modin.readthedocs.io/en/latest/development/using_pandas_on_ray.html)
- [PandasOnDask](https://modin.readthedocs.io/en/latest/development/using_pandas_on_dask.html)

```

## Creating a development environment

To get required dependencies for these Jupyter Notebooks
you should create a development environment with `pip`
using `requirements.txt` file located in the respective directory:

```bash
pip install -r execution/pandas_on_ray/requirements.txt
```

> to install dependencies needed to run notebooks using Ray engine or

Reviewer: ```suggestion
to install dependencies needed to run notebooks with Modin on `PandasOnRay` execution or
```

```bash
pip install -r execution/pandas_on_dask/requirements.txt
```

> to install dependencies needed to run notebooks using Dask engine

Reviewer: ```suggestion
to install dependencies needed to run notebooks with Modin on `PandasOnDask` execution.
```

> ## Run Jupyter Notebooks

Reviewer: Why was this file moved? We should have a single one for pandas_on_ray and another one for pandas_on_dask in the respective directories. Remember about the OmniSci engine.

Author: It was moved to this directory to make it more convenient to user where to start jupyter notebooks - we should start notebooks in the directory where images are located and if we move readme closer to notebooks we will need to tell something like "go in the parent directory, then once again and only then you should start jupyter server". Additionally, if we place readme here we will be capable to avoid bunch of code duplication i believe (for OmniSci specific notes separate section can be created).

Reviewer: This file should be corrected with respect to different exexutions.

Author: Right, notes for different executions are added.

> A Jupyter Notebook server example can be run from the current directory as follows:

Reviewer: ```suggestion
A Jupyter Notebook server can be run from the current directory as follows:
```
?

```bash
jupyter notebook
```

> And then navigate to the needed notebook (for example to the `execution/pandas_on_ray/local/exercise_1.ipynb`).

Author: Jupyter server should be started in the directory containing `img` directory, otherwise notebooks will not capable to access images.

Reviewer: Can we use absolute paths, but not relative?

Author: Fixed.

Reviewer: I see the relative paths. Can we use absolute paths instead?

Author: Absolute patrh `examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb` is used now.

> Navigate to a concrete notebook (for example to the `examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb`).

Reviewer: ```suggestion
Navigate to a concrete notebook (for example, to the `execution/pandas_on_ray/local/exercise_1.ipynb`).
```
Since a jupyter server is run from the current directory we should use relative paths to notebooks.

> And then navigate to the needed notebook (for example to the `examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb`).

Reviewer: ```suggestion
Navigate to a concrete notebook (for example to the `examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb`).
```
